src/ngraph/builder/quantization.cpp

```diff
@@ -33,6 +34,34 @@ namespace ngraph
 {
     namespace builder
     {
+        std::shared_ptr<Node> ScaledQuantize(std::shared_ptr<Node> input,
+                                             std::shared_ptr<Node> min,
+                                             std::shared_ptr<Node> max,
+                                             const ngraph::element::Type& type,
+                                             const ngraph::AxisSet& axes,
+                                             op::Quantize::RoundMode round_mode)
+        {
+            float scale =
+                builder::quantization_util::get_quantize_scale(min, max, type.is_signed());
+            auto offset = op::Constant::create(element::f32, Shape{1}, {0});
```

Comment:
get_quantize_scale accepts 2 nodes, typecasts them to const node, reads data from them, and computes float scaleoffset. That is a slightly mixed api because it accepts nodes and outputs float.

a) get_quantize_scale should accept vectors or floats and spit out floats (so its a function that does not accept ngraph nodes)
or
b) get_quantize_scale accepts min-max ngraph nodes and returns scale-offset ngraph nodes.

I prefer approach (b) because: for now we just accept min-max as consts and compute the scale offset in c++ code. the problem is that the quantize ops must declare min-max as static inputs (must be const or inputs to encapsulates, that they cannot be in the middle of the encapsulate or in the middle of an ngraph graph)
later we could have ngraph nodes compute scale-offset from min-max nodes. In that case option (b) would create the subgraph that converts min-max to scale-offset nodes.